src/locales/de/translation.json

```diff
@@ -50,7 +50,7 @@
   "Imports per day": "Importe pro Tag",
   "Infectious": "Infektiös",
   "Infectious period": "Infektiöser Zeitraum",
-  "Initial suspected cases": "Erste Verdachtsfälle",
+  "Initial number of cases": "Erste Verdachtsfälle",
```

Comment:
Some of these translations seem a bit outdated now. For example "Anfänglich Infizierte" or so would be more appropriate now.